### ISSUE TITLE
Test a trained model with nopbc data calculate also virial error

### ISSUE THREAD

**Issue by TinacciL** [deepmodeling/deepmd-kit#1286]
I installed via Docker the 2.0.3 deepmd-kit version for GPU.
I test also the same version with the CPU version installed via conda.

When I test the trained model the code calculated also the viral error but I test and trained the model only without pbc data.

```
[0] DEEPMD INFO    Virial RMSE        : 6.379999e+00 eV
[0] DEEPMD INFO    Virial RMSE/Natoms : 3.322916e-02 eV
```

**Comment by wanghan-iapcm** [comment]
Hello, in this case the virial test does not make sense, one may safely ignore them.

**Comment by njzjz** [comment]
Fixed in #1314.
